homeassistant/components/daybetter_services/tests/conftest.py

```diff
@@ -0,0 +1,49 @@
+"""Configuration for pytest."""
```

Comment:
The tests are in the wrong place. They should be at the root of the project. See other integrations for examples.